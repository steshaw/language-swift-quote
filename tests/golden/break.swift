var n = 1
while true {
  if (n < 1) { break }
  n += 1
}
