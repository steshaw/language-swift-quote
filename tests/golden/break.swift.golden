"<stdin>" (line 3, column 11):
unexpected "1"
expecting identifier